Lib/idlelib/idle_test/template.py

```diff
@@ -0,0 +1,23 @@
+"Test , coverage %."
+
+import unittest
+from test.support import requires
+from tkinter import Tk
+
+import
```

Comment:
Typo?

Reply:
I don't see a misspelling. The name of the tested module goes after `Test ` above and after `import`.  But I can see that might not be clear if you did not write it ;-).

Reply:
So let me make sure that I understand, you have `import` as a placeholder for someone writing a test for a module (say "xyz"). Folks would copy the template, change the file's name to `test_xyz.py`. Inside the file a person writing tests would insert `xyz` as:

```
Test xyz coverage %.
...
import xyz
```

Just clarifying that an `import` statement without a module name isn't planned to be executed. (If I'm missing the boat, and you can execute `import` without a `.` or module name, please let me know since I didn't think one could do this).

Reply:
Perhaps it would be clearer to name the file `test_modname.py.template`, and insert `modname` where the module name should go rather than leaving blanks.  From a UNIX perspective, this would allow one to do `sed 's/modname/real_mod_name/g' test_modname.py.template > test_real_mod_name.py` to create the test file from the template in one step.

Reply:
Windows does not have sed, and for two insertions, it is not worth the trouble.  The name 'template.py' puts the file just above the first test_a.py file, making it initially visible in IDLE's open dialog and easy to find, and automatically 'saved as' a .py file.

I am the initial user of the file, for this issue.  It standardizes the header and footer lines and documents some of the setup and teardown needed for gui tests.  (I just discovered and added a new trick for  EditorWindow.)  I intend to discuss it, along with other gui-testing tricks not part of the template, in the README.